### ISSUE TITLE
Is it possible to modify thread title? 

### ISSUE THREAD

**Issue by Songyu-Wang** [quip/quip-api#89]
Is there a way to do it? I have trouble finding it in the API doc.
Thanks!

**Comment by jesseditson** [comment]
Yes! The title is inferred from the first H1 in the document, so you can just change that. To do so, I'd reccomend:

- use `/threads/:id` to get the thread content
- you can see the section ID of the title is set as the `id` field of the first H1:
![image](https://user-images.githubusercontent.com/370239/114450758-3fea2100-9b8b-11eb-96e3-054544dc4878.png)

- You can then POST to `/threads/edit-document` to edit the title using the params:

`content`: the new title
`thread_id`: the thread ID
`section_id`: the ID of the title as you retrieved in the last step
`location`: 4 (meaning replace section content)

More documentation on the edit document endpoint can be found here:

https://quip.com/dev/automation/documentation#edit-a-document

Hopefully this helps!

**Comment by Songyu-Wang** [comment]
@jesseditson thank you for your reply. Yes, it is true that is can modify the title in the main doc body. However, the title field above the tool bars does not get updated with it. This is the issue I was trying to raise

**Comment by jesseditson** [comment]
Ah you're right, this is supposed to track but it looks like via the API it doesn't (when I edit it manually it does).

**Comment by jesseditson** [comment]
I don't actually see any way in our current code to do this, so I'll leave this open as a feature request!

**Comment by Qureshima** [comment]
Hi @jesseditson -san
My name is Keiichi Kureshima.
I was a Solution Engineer and Quip Evangelist at Salesforce Japan until last year, promoting Quip in the Japanese market.

I would like to discuss one thing with you.

There are many cases where Quip document naming rules are in place and search accuracy is low.
In Japanese, it is not common to put spaces between words in a sentence. Therefore, words are often connected to each other to write the title sentence. As a result, search terms are often not found.

To solve this problem, we are developing LiveApps, which can modify title strings in a batch for a list of title search results, but we are unable to do so due to a bug in the API.

Is it possible to raise the priority of addressing this bug?

![image](https://user-images.githubusercontent.com/91104865/186792633-fdd6a8ff-6f34-449b-bd80-e6ecd1702cf5.png)

**Comment by jesseditson** [comment]
Hello there @Qureshima ! I'm sorry to say that I no longer am working at Quip, so I cannot help here! I am also unsure who is the current owner of Live Apps, so I can't help route you either - I'm sorry, good luck!

**Comment by Qureshima** [comment]
@jesseditson 
Thanks for the reply!
I understood the situation. I'll take another route to check it out!
